crates/sui-framework/packages/sui-framework/sources/coin.move

```diff
@@ -162,6 +162,21 @@ public fun put<T>(balance: &mut Balance<T>, coin: Coin<T>) {
     balance.join(into_balance(coin));
 }
 
+// === Address Balance <-> Coin utility functions ===
+
+/// Redeem a `Withdrawal<Balance<T>>` and create a `Coin<T>` from the withdrawn Balance<T>.
+public fun redeem_funds<T>(
```

Comment:
Do we really need these delegates?

Reply:
The plan long term is to not have any commands on the PTB side for such use cases, we will just auto-convert... though that auto conversion might call these :P so we can leave them in I suppose